src/main/app/case/CaseApi.ts

```diff
@@ -71,16 +89,29 @@ export class CaseApi {
     }
   }
 
-  public async triggerEvent(caseId: string, userData: Partial<Case>, eventName: string): Promise<CaseWithId> {
+  public async saveUserData(caseId: string, userData: Partial<Case>, eventName = CITIZEN_UPDATE): Promise<CaseWithId> {
```

Comment:
not really a fan of this, it doesn't seem like a good abstraction but happy to live with it if you are

Reply:
i agree

Reply:
The main reason I separated these out was because it modifying the user data via `toApiFormat`, when this is moved to the PostController then I can revert this change.